test/hotspot/jtreg/containers/docker/TestCPUAwareness.java

```diff
@@ -70,10 +70,14 @@ public static void main(String[] args) throws Exception {
             testActiveProcessorCount(2, 2);
 
             // cpu quota and period
-            testCpuQuotaAndPeriod(50*1000, 100*1000);
-            testCpuQuotaAndPeriod(100*1000, 100*1000);
-            testCpuQuotaAndPeriod(150*1000, 100*1000);
-            testCpuQuotaAndPeriod(400*1000, 100*1000);
+            testCpuQuotaAndPeriod(50*1000, 100*1000, false);
+            testCpuQuotaAndPeriod(100*1000, 100*1000, false);
+            testCpuQuotaAndPeriod(150*1000, 100*1000, false);
+            testCpuQuotaAndPeriod(400*1000, 100*1000, false);
+            testCpuQuotaAndPeriod(50*1000, 100*1000, true);
+            testCpuQuotaAndPeriod(100*1000, 100*1000, true);
+            testCpuQuotaAndPeriod(150*1000, 100*1000, true);
+            testCpuQuotaAndPeriod(400*1000, 100*1000, true);
```

Comment:
It should be sufficient to test the `true` case once. I.e. one test `testCpuQuotaAndPeriod(50*1000, 100*1000, true);` should be enough (as this would extend container test runs needlessly).

Reply:
done